.buildkite/pipeline.release.yml

```diff
@@ -10,14 +10,13 @@ steps:
     artifact_paths:
       - dist/**/*
     plugins:
-      - docker-compose#v5.2.0:
+      - docker-compose#v5.10.0:
```

Comment:
You've merged this, but it would be good to add the recently introduced `progress: plain` option too